src/templates/rest_framework/api.html

```diff
@@ -1,4 +1,320 @@
-<html lang=nl>
-{% extends "rest_framework/base.html" %}
-</html>
+<!-- 
```

Comment:
Jammer dat je de hele template moet kopieren, maar, weinig keus denk ik, als je de base template aan wilt passen.

Reply:
Ja klopt, erg jammer. Er zijn veel opties in Django om delen uit de base.html te customizen maar geen default mogelijkheid voor de HTML tag en lang attribuut.